pyfar/dsp/dsp.py

```diff
@@ -995,18 +995,26 @@ def pad_zeros(signal, pad_width, mode='after'):
     --------
     >>> import pyfar as pf
     >>> impulse = pf.signals.impulse(512, amplitude=1)
-    >>> impulse_padded = pf.dsp.pad_zeros(impulse, 128, mode='after')
+    >>> impulse_padded = pf.dsp.pad_zeros(impulse, 128, mode='end')
 
     """
 
     if not isinstance(signal, pyfar.Signal):
         raise TypeError('Input data has to be of type: Signal.')
 
+    if mode in ['before', 'after']:
+        warnings.simplefilter('always')
+        warnings.warn(('Mode "before" and "after" will be renamed into '
+                       '"beginning" and "end" and can no longer be used in '
+                       'Pyfar 0.5.8.'), PendingDeprecationWarning)
```

Comment:
This would be released in 0.6.0 and deprecated two minor releases later:
```suggestion
                       'Pyfar 0.8.0.'), PendingDeprecationWarning)
```